.github/workflows/check-all.yml

```diff
@@ -0,0 +1,18 @@
+name: Check All
```

Comment:
can we just all `test.yml`? 

This is just running test and not check-all

Reply:
+1